tests/positive/Format.juvix

```diff
@@ -375,3 +375,15 @@ l4 : List (List Int) :=
   ];
 
 -- Comment at the end of a module
```

Comment:
Move this line (`-- Comment at the end of a module`) to the end of the module.

Reply:
Done